sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -420,63 +425,71 @@ trait CheckAnalysis extends PredicateHelper {
             }
 
           case alter: AlterTable if alter.childrenResolved =>
```

Comment:
nit: `case alter: AlterTable if alter.childrenResolved && alter.table.isInstanceOf[ResolvedTable]`

Then we don't need to change the indentation below and save a lot of code diff.

Reply:
BTW we can't hit v1 table here right? We should have converted alter table command to v1 if the table is v1, before we reach here.

Reply:
Thanks, changed as suggested.

That's correct. If a table is `V1Table`, the alter table are converted to v1 in `ResolveSessionCatalog`.